admin/c2cgeoportal_admin/views/layers_wmts.py

```diff
@@ -10,12 +12,14 @@
 from c2cgeoform.views.abstract_views import ListField, ItemAction
 
 from c2cgeoportal_admin import _
-from c2cgeoportal_commons.models.main import LayerWMTS, LayerWMS, TreeItem
 from c2cgeoportal_admin.schemas.dimensions import dimensions_schema_node
+from c2cgeoportal_commons.models.main import \
+    LayerWMTS, LayerWMS, TreeGroup, LayergroupTreeitem, LayerGroup, TreeItem
+from c2cgeoportal_admin.views.dimension_layers import DimensionLayerViews
 from c2cgeoportal_admin.schemas.metadata import metadatas_schema_node
 from c2cgeoportal_admin.schemas.interfaces import interfaces_schema_node
 from c2cgeoportal_admin.schemas.restrictionareas import restrictionareas_schema_node
-from c2cgeoportal_admin.views.dimension_layers import DimensionLayerViews
+from c2cgeoportal_admin.schemas.treegroup import base_deferred_parent_id_validator
```

Comment:
Why you do move imports that are in alphabetic order ?

Reply:
Probably when  I resolved the merge conflicts